src/common/combinator.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2014-2018, The Monero Project
```

Comment:
That file doesn't seem to have any old code, so 2018

Reply:
ok, will do it